test/unit/spec/nbextensions/bulkImportCell/bulkImportCell-spec.js

```diff
@@ -593,21 +630,12 @@ define([
                 expect(bulkImportCellInstance.jobManager.initBatchJob).toHaveBeenCalledTimes(1);
                 const batchId = JobsData.batchParentJob.job_id;
                 // bus calls to init jobs, request info, cancel jobs, and stop updates
-                const callArgs = JobsData.allJobsWithBatchParent
-                    .map((jobState) => {
-                        return ['request-job-updates-start', { jobId: jobState.job_id }];
-                    })
-                    .concat(
-                        [['request-job-status', { batchId }]],
-                        [['request-job-info', { batchId }]],
-                        [['request-job-cancel', { jobIdList: [batchId] }]]
-                    )
-                    .concat(
-                        JobsData.allJobsWithBatchParent.map((jobState) => {
-                            return ['request-job-updates-stop', { jobId: jobState.job_id }];
-                        })
-                    );
-
+                const callArgs = [
+                    ['request-job-updates-start', { batchId }],
+                    ['request-job-info', { batchId }],
+                    ['request-job-cancel', { jobIdList: [batchId] }],
+                    ['request-job-updates-stop', { batchId }],
```

Comment:
All those nice new endpoints, thanks to @n1mus!

Reply:
👏